components/notifier/src/quality_time_notifier.py

```diff
@@ -23,18 +23,32 @@ async def notify(log_level: int = None) -> None:
             response = await session.get(f"http://{server_host}:{server_port}/api/v3/reports")
             json = await response.json()
 
-        notifications = reds_per_report(json)
+        notifications = reds_that_are_new(json)
 
+        total_new_red_metrics = 0
         for notification in notifications:
```

Comment:
```suggestion
        total_new_red_metrics = sum(notification["new_red_metrics"] for notification in notifications)
```
And line 30 can be removed.